tests/integration/run_tests.rs

```diff
@@ -4732,12 +4732,12 @@ fn permission_prompt_strips_ansi_codes_and_control_chars() {
       );
     // will be uppercase on windows
     let env_name = if cfg!(windows) {
-      "DO YOU LIKE ICE CREAM? Y/N"
```

Comment:
Nit: we have to update the test name at L4728 https://github.com/denoland/deno/pull/22468/files#diff-cb22558f2ba9727b76ba410dcee434ec28c95f9198ebe736842b7270638ee229L4728